plat/common/aarch64/plat_common.c

```diff
@@ -56,11 +56,13 @@ void bl32_plat_enable_mmu(uint32_t flags)
 
 void bl31_plat_runtime_setup(void)
 {
+#if !DEBUG
```

Comment:
OK, I will implement my own bl31_plat_runtime_setup().  Thanks!
Please discard this pull request.